### DISCUSSION TITLE
KeyboardAwareScrollView not working

### DISCUSSION THREAD

**Discussion by IAlphaOmegaI** [kirillzyusko/react-native-keyboard-controller#713] [General]
here is the code sample:
 <KeyboardAwareScrollView
      style={{
          flex: 1,
        }}
      >
        {new Array(10).fill(0).map((_, i) => (
          <TextInput
            key={i}
            placeholder={`TextInput#${i}`}
            className="h-20"
            keyboardType={i % 2 === 0 ? "numeric" : "default"}
          />
        ))}
      </KeyboardAwareScrollView>

**Comment by kirillzyusko** [comment]
@IAlphaOmegaI please, provide a minimal reprpoduction example. Information from the code snippet is not enough as I don't have an information about `TextInput` styling and can not copy/paste it into my project.

**Reply by IAlphaOmegaI** [reply]
React's Native TextInput with nativewind for classnames (what would be equivalent to {height: 80})

**Reply by kirillzyusko** [reply]
@IAlphaOmegaI can you please create a small reproducible example so that I can clone it and run on my machine to check where is the issue?

A similar code works in example project without any issues so I should have a real project where I can debug a problem 🙂
